docs/cpp/config-clang-mac.md

```diff
@@ -3,124 +3,143 @@ Order: 4
+The `code .` command opens VS Code in the current working folder, which becomes your "workspace". As you go through the tutorial, you will create three files in a `.vscode` folder in the workspace:
 
-- `c_cpp_properties.json` (compiler path and IntelliSense settings)
-- `tasks.json` (build instructions)
+- `tasks.json` (compiler build settings)
 - `launch.json` (debugger settings)
+- `c_cpp_properties.json` (compiler path and IntelliSense settings)
 
-To reuse the configuration, you can copy these files to a new workspace and adjust the program name and other settings as needed.
+### Add hello world source code file
 
-## Configure the compiler path
+In the File Explorer title bar, select **New File** and name the file `helloworld.cpp`.
 
-1. Press `kb(workbench.action.showCommands)` to open the Command Palette. It looks like this:
+![New File title bar button](images/msvc/new-file-button.png)
 
-   ![Command Palette](images/clang-mac/mac-command-palette-configurations.png)
+Paste in the following source code:
 
-1. Start typing "C/C++" and then choose **Edit Configurations (UI)** from the list of suggestions. This opens the **C/C++ Configurations** page. When you make changes here, VS Code writes them to a file called `c_cpp_properties.json` in the .vscode folder.
+```cpp
+#include <iostream>
+#include <vector>
+#include <string>
 
-   ![Command Palette](images/clang-mac/intellisense-configurations-mac-clang.png)
+using namespace std;
 
-1. Find the **Compiler path** setting. VS Code will attempt to populate it with a default compiler based on what it finds on your system. For Clang on macOS, the path should look like this: `/usr/bin/clang`.
+int main()
+{
+    vector<string> msg {"Hello", "C++", "World", "from", "VS Code", "and the C++ extension!"};
 
-   The **Compiler path** setting is the most important setting in your configuration. The extension uses it to infer the path to the C++ standard library header files. When the extension knows where to find those files, it can provide lots of useful information to you as you write code. This information is called *IntelliSense* and you'll see some examples later in this tutorial.
+    for (const string& word : msg)
+    {
+        cout << word << " ";
+    }
+    cout << endl;
+}
+```
 
-1. Set **IntelliSense mode** to `${default}`, which on macOS is `clang-x64`.
+Now press `kb(workbench.action.files.save)` to save the file. Notice that your files are listed in the **File Explorer** view (`kb(workbench.view.explorer)`) in the side bar of VS Code:
 
-1. You only need to modify the **Include path** setting if your program includes header files that are not in your workspace or in the standard library path.
+![File Explorer](images/msvc/file-explorer.png)
```

Comment:
This screenshot also shows the old Debug/Run icon. Fine to do later.

Reply:
Fixed by removing the portion that had the debug button in view. Fixes for whomever else is using this file.